src/release/triage-procedure.md

```diff
@@ -30,13 +40,93 @@
+[s-inactive]: https://github.com/rust-lang/rust/pulls?q=is%3Aopen+is%3Apr+label%3AS-inactive+sort%3Aupdated-asc
 [s-blocked-closed]: https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3AS-blocked-closed+sort%3Aupdated-asc
 [s-inactive-closed]: https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3AS-inactive-closed+sort%3Aupdated-asc
+[s-experimental]: https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3AS-experimental+sort%3Aupdated-asc
+[no-status-tags]: https://github.com/rust-lang/rust/pulls?q=is%3Apr+is%3Aopen+-label%3AS-waiting-on-author+-label%3AS-waiting-on-review+-label%3AS-waiting-on-team+-label%3AS-waiting-on-bors+-label%3AS-waiting-on-crater+-label%3AS-waiting-on-bikeshed+-label%3AS-waiting-on-perf+-label%3AS-blocked+-label%3AS-inactive+-label%3AS-blocked-closed+-label%3AS-inactive-closed+-label%3AS-waiting-on-fcp+-label%3AS-experimental
 [crater]: https://github.com/rust-lang-nursery/crater
 [bors]: https://github.com/rust-lang/homu
 
 ### Procedure
 
+We primarily triage three status labels: S-waiting-on-review,
+S-waiting-on-author, and (once in a while) S-blocked. Here is the procedure for
+each:
+
+#### S-waiting-on-review
+
+Click [this link][S-waiting-on-review] to see all PRs with the
+S-waiting-on-review label. Only triage PRs that were last updated 15 days or
+more ago (give or take a day).
+
+For each PR:
+
+1. **If** the PR has new conflicts, CI failed, or a new review has been made
+   **then** change the label to S-waiting-on-author and ping the author.
+
+2. Add the PR to your [report].
+
+#### S-waiting-on-author
+
+Click [this link][S-waiting-on-author] to see all PRs with the
+S-waiting-on-author label. Only triage PRs that were last updated 15 days or
+more ago (give or take a day).
+
+For each PR:
+
+1. **If** the author did what the PR was waiting on them for **then** update the
+   label to S-waiting-on-review.
+
+2. **If** the author is a member of a Rust team (not working groups — teams like
+   T-compiler, T-lang, T-rustdoc, etc.) **then** don't ping anyone.
+
+   **Otherwise,** ping the author.
+
+3. Add the PR to your [report].
+
+#### S-blocked
+
+You only need to check S-blocked PRs occasionally (e.g., once a month).  Click
+[this link][S-blocked] to see all PRs with the S-blocked label.
+
+For each PR:
+
+1. **If** it is still blocked **then** leave it as-is.
+
+   **Otherwise,** if it is no longer blocked, then remove S-blocked (and add a
+   status label like S-waiting-on-review if appropriate).
+
+2. Add the PR to your [report].
+
+#### Triage Report
+[report]: #triage-report
+
+You should record information about each PR you triage in a report. The report
+is just a small document that looks like:
+
+> #### S-waiting-on-review
+>
+> #12345 20 days - still waiting on review - author: ferris, assignee: bors
+>
+> \[...\]
+
+However, each person has a different format for their triage reports, so yours
```

Comment:
this doesn't need to be mentioned. Instead mention that format isnt important as long as the following information is conveyed:
1) pr number, preferably linked
2) author/reviewer and who it is waiting on (or the team it is waiting on )
3) status of it and if possible the last activity 
4) days since last active (author, reviewer or a team member has pushed/reviewed/commented on it